module/data/activity/attack-data.mjs

```diff
@@ -84,8 +84,9 @@ export default class AttackActivityData extends BaseActivityData {
   /** @inheritDoc */
   get activationLabels() {
     const labels = super.activationLabels;
-    if ( labels && (this.item.type === "weapon") && this.item.labels?.range && !this.range.override ) {
-      labels.range = this.item.labels.range;
+    if ( labels && (this.item.type === "weapon") && !this.range.override ) {
+      if ( this.item.labels?.range ) labels.range = this.item.labels.range;
+      if ( this.item.labels?.reach ) labels.reach = this.item.labels.reach;
```

Comment:
We might want to clarify that one of these is reach, because a natural weapon with range just displays two unlabeled lengths (this has a 15 ft range and the default 5 ft reach):

<img width="190" alt="Screenshot 2025-02-16 at 10 20 47" src="https://github.com/user-attachments/assets/ce0c2a19-7008-4e63-82f4-bcfad00074e6" />